### PR TITLE
Adding 10 open source blockchain projects to contribute to

### PR DESCRIPTION
**Pull request by SaadAnsari17**

### CODE REVIEW

File: Blockchain/Contribute/README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 ## Open Source Projects you can contribute to
-
+-[Aelf](https://github.com/AElfProject/AElf)
```
Comment (kaiwalyakoparkar) [inline Blockchain/Contribute/README.md:2]: Please add space between - and [...]
```diff
- -[Aelf]
+ - [Aelf]
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: Blockchain/Contribute/README.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
 ## Open Source Projects you can contribute to
```
Comment (kaiwalyakoparkar) [inline Blockchain/Contribute/README.md:1]: Add a blank line below the title text

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: Blockchain/Contribute/README.md (hunk 2 above)
Comment (SaadAnsari17) [reply Blockchain/Contribute/README.md:1]: Done👍

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**